### PR TITLE
Docker development utility

### PR DESCRIPTION
**Pull request by josiahdavis**
*Issue #, if available:* NA

*Description of changes:*

Creation of a utility: stand-alone Dockerfile for interactive data science development:

- amazon linux 2 base
- python version management with miniconda
- mount your current directory and aws credential directory
- AWS CDK installation
- zsh shell
- pip (or conda) install python packages

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: modules/environment/util/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        openssl-devel \
+        zlib-devel \
+        readline-devel \
+        git \
+        blas blas-devel \
+        lapack lapack-devel \
+        wget \
+        zsh \
+        which \
+        tmux
```
Comment (yihyap) [inline modules/environment/util/docker/Dockerfile:17]: Include `tar` package required for vscode remote dev

**Review by yihyap — COMMENTED** [review]
Add in a python package to support vscode remote development on container.

File: modules/environment/util/docker/Dockerfile (hunk 1 above)
Comment (josiahdavis) [reply modules/environment/util/docker/Dockerfile:17]: > Add in a python package to support vscode remote development on container.

Which package are you referring to? tar or something else?

**Comment by josiahdavis** [comment]
@yihyap added tar install, lmk if any other comments.

**Comment by yihyap** [comment]
Good to go.

**Pull request merged**